google/cloud/security/inventory/pipeline_builder.py

```diff
@@ -49,9 +50,53 @@ def __init__(self, cycle_timestamp, config_path, flags,
         self.flags = flags
         self.api_map = api_map
         self.dao_map = dao_map
+        self.initialized_api_map = {}
```

Comment:
nit, maybe just default this to {} and make it a kwarg? 

`.... api_map, dao_map, initialized_api_map={}):`